### PR TITLE
Revert "Add some fixes and user option"

### PR DESCRIPTION
**Pull request by VishnoiAman777**
Hey there is some error in variables in the main.py file. Kindly review it please @sky-2002

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -14,6 +13,4 @@
 
     if usr_input == 1:
         task_input = input("Enter the task details")
-        todo.add_task(task_input)
-    elif user_input==2:
```
Comment (sky-2002) [inline main.py:18]: I am changing this

File: task.py [hunk 2]
```diff
@@ -19,6 +19,7 @@ def delete_task(self, task_id):
     def mark_complete(self, task_id):
         pass
 
-    def list_task(self):
-        for i, task in enumerate(self.task_details, start=1):
-            print(f"{i}. {task}")
\ No newline at end of file
+    def list_task():
+        for i, task in enumerate(tasks, start=1):
```
Comment (sky-2002) [inline task.py]: Also need to fix this

Comment (VishnoiAman777) [reply task.py]: Yes this looks good now sounds like you are understanding it

File: main.py (hunk 1 above)
Comment (VishnoiAman777) [reply main.py:18]: Yes this is it

**Comment by sky-2002** [comment]
Also, I am changing the `delete_task` function.

**Pull request merged**